pypeit/utils.py

```diff
@@ -18,7 +18,8 @@
 from pypeit.core import pydl
 from pypeit import msgs
 
-def wavegrid(wave_min, wave_max, dwave, osamp=1.0):
+
+def wavegrid(wave_min, wave_max, dwave, samp_fact=1.0):
```

Comment:
seems like this should go in a method in `wavecal`

Reply:
Yeah perhaps to wavecal/wvutils.py.

Reply:
done